test/controllers/update/overseas.entity.review.controller.spec.ts

```diff
@@ -43,6 +45,8 @@ mockCompanyAuthenticationMiddleware.mockImplementation((req: Request, res: Respo
 const mockServiceAvailabilityMiddleware = serviceAvailabilityMiddleware as jest.Mock;
 mockServiceAvailabilityMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
 
+const mockGetCompanyPsc = getCompanyPsc as jest.Mock;
```

Comment:
mock to return empty list?

Reply:
updated tests in test/controllers/update/update.beneficial.owner.type.controller.spec.ts